.gitignore

```diff
@@ -10,6 +10,7 @@ firmware*
 credentials.h
 node_modules
 code/utils
+code/package-lock.json
```

Comment:
package-lock.json file is not supposed to be gitignored, you need to commit it to the repository.

when installing packages with npm install, newer versions of npm(starting from v5) generate the lock file to ensure that transitive dependencies are resolved to exact the same versions on different computers.